src/main/java/org/example/homework5/GenerateSortPrintArray.java

```diff
@@ -0,0 +1,49 @@
+package org.example.homework5;
+
+import java.util.Random;
+
+public class GenerateSortPrintArray {
+    public static void main(String[] args) {
+        int[] numbers = getIntegers(10);
+        System.out.println("Here is your randomized unsorted array: \n");
+        printArray(numbers);
+        int[] sortedNumbers = sortIntegers(numbers);
+        System.out.println("\nHere is your sorted array:\n");
+        printArray(sortedNumbers);
+
+    }
+
+    private static int[] getIntegers(int size) {
+        Random random = new Random();
+        int[] arr = new int[size];
+        for (int i = 0; i < arr.length; i++) {
+            arr[i] = random.nextInt(100);
+        }
+        return arr;
+    }
+
+    private static void printArray(int[] numbers) {
+        int element = 0;
+        for (int i : numbers) {
+            element++;
+            System.out.println("Element " + element + " contents: " + i);
+        }
+
+    }
+
+    private static int[] sortIntegers(int[] numbers) {
+
```

Comment:
in previous methods there were no empty lines after the method declaration line, please keep the code in similar style